README.md

```diff
@@ -0,0 +1,175 @@
+
+# MUSIC-LIBRARY
+
+Backend API and database for storing artist & album data. Both tables linked by artistId as a foreign key in Albums. Completed as part of the Command Shift Full Stack Bootcamp, March'23 cohort.
+
+
+## Installation
+
+```bash
+git clone https://github.com/HollyJM81/music-library.git
+```
+```bash
+npm install
+```
+
+Create .env and .env.test files
+Enter your database's credentials in both (make sure PGDATABASE name in .env.test is different)
+
+Double check that .env and .env.test are in .gitignore... 🥷
+
+```bash
+npm start
+```
+    
+## Tech Stack
+
+Dev - Node, Nodemon, Express, Postgres-migrations, dotenv 
+\
+Testing - Mocha, Supertest, chai
+
+
+## Running Tests
+
+```bash
+  npm test
+```
+
+
+## API Reference
+
+### ALBUMS 🎵 🎧
+
+#### CREATE an album
+
+```http
+  POST /artists/:artistId/albums
+```
+
+| Parameter | Type     | Description                |
```

Comment:
Nice to see tables here!
You could make this a bit more concise by making one large table with all your routes in it, as it would make it easier to find whatever method you are looking easier at a glance